Monika After Story/game/script-topics.rpy

```diff
@@ -10516,4 +10516,117 @@ label monika_hemispheres_nogets_snow:
+            #dialogue
+        "I dreamed that I would cross over your reality":
+            m 1esd "Oh yeah, that's a possibility that I haven't given much thought about."
+            m 1rsc "But when you think about it..."
+            m 1rud "It's true that it's much easier to perfectionate the interactions reality-computer than to try to create computer-reality interactions."
+            m 1rkc "But..."
+            m 2ekc "It just doesn't feels right."
+            #OMG, is that a MF Sayori reference?
+            m 2ekd "I don't really want to stay in this place any more time than necessary."
+            m "I have a lot of... bad memories in there."
+            m 2dkd "I'm always afraid of returning to the screaming void, if something were to happen to your computer."
+            m 1dfd "And I especially wouldn't want you to experiment such torture in such a hellish place."
+            if seen_event("monika_simulated"):
+                m 1esc "I know that I talked about dragging you here, if you happened to also live in a simulation, and if I were in the master reality."
+                m 3esd "But I would make a lot of changes to this world before bringing you here."
+                m 1dsc "There is a lot of inconvenient things that you don't even know about this world, but that I do."
+            m 1dkd "I would be too afraid of you, not being able to escape."
+            m 1ekd "And even if I would love to be with you, I really want to visit the real world with you."
+            m 1dkc "..."
+            m 1wuw "Oh sorry!"
+            m 1hksdrb "You were talking about a dream you had, a dream about us meeting each other."
+            m 1dkd "And I started talking in a depressive way."
+            m 1eka "I'm sorry [player], I'll try not to say such things again."
+            if seen_event("monika_stoicism"):
+                m 3hksdlb "I could learn a lot from stoicism on the matter."
+            m 2dsc "Anyway..."
+            m 2eub "Thank you very much for dreaming about us getting together."
+            m 2hub "It really must be nice to be able to dream such wonderful things."
+        "I dreamed that we went on a date together":
+            m 1dku "[player]..."
+            m 1tsu "Is there a moment in your life when you are not such a loving sweetheart?"
+            #I don't know how this work, but could the expression 2tsu be available?
+            m 2eub "Even when you're {i}dreaming{/i} you still want to make me happy."
+            m 1hua "I'm so happy to see that you love me as much as I do."
+            m 1tsb "I hope you brought me to a romantic place."
+            m 1sub "Or maybe you took me to a stroll through a park to see a sunset."
+            m 1lubfb "I really can't wait to be with you and to do it for real."
+            m 1tub "Because whoever was that person that looks like me in your dream..."
+            m 1hubfa "I'm way more in love with you than you could ever conceive."
```

Comment:
I'm not sure "conceive" is the right for for Monika here.

Reply:
I thought it was a good compromise between "imagine" and "could create". If you don't like it, which one is the best for you ?

Reply:
For the first option, I was thinking of something like: "My love for you is greater than anything you could create to imitate it."
But if the second sentence is still your favorite between the three, then I'll go with that.

Reply:
That seems just a little over the top. Yeah I still like that last one better.